treeherder/webapp/api/runnable_jobs.py

```diff
@@ -57,6 +76,40 @@ def list(self, request, project):
                 'state': 'runnable',
                 'result': 'runnable'})
 
+        for label, node in tc_graph.iteritems():
+            build_platform = node['task']['extra']['treeherder']['build']['platform']
+            job_type_name = node['task']['metadata']['name']
+
+            # Not all tasks have a group name
+            if 'groupName' in node['task']['extra']['treeherder']:
+                job_group_name = node['task']['extra']['treeherder']['groupName']
+            else:
+                job_group_name = ""
+
+            ret.append({
+                'build_platform_id': "",
```

Comment:
do you even need to set the ``_id`` fields in this struct?  I suppose you're just being consistent with the BuildBot side?  If the UI doesn't reference them, we could just omit them from the payload entirely.  :)